src/mxbase.jl

```diff
@@ -41,6 +41,12 @@ function get_paths()
             error("The MATLAB path is invalid. Set the MATLAB_HOME evironmental variable to the MATLAB root.")
         end
         default_startcmd = "exec $(Base.shell_escape(default_startcmd)) -nosplash"
+
+        # Disable DISPLAY if asked or not avaible
+        if get(ENV, "DISPLAY", nothing) == nothing ||
+            get(ENV, "MATLAB_NO_DISPLAY", nothing) != nothing
+            default_startcmd *= " -nodisplay -nojvm"
```

Comment:
No sure about that, if you leave out the `-nojvm`, matlab will still call opengl and other display stuff (at least under 2014), so using only `nodisplay` doesn't realy solve the issue. Can you reproduce this behaviour?